src/main/java/frc/trigon/robot/subsystems/swerve/SwerveModuleConstants.java

```diff
@@ -18,46 +23,54 @@ public class SwerveModuleConstants {
             DRIVE_MOTOR_INVERTED_VALUE = InvertedValue.CounterClockwise_Positive,
             STEER_MOTOR_INVERTED_VALUE = InvertedValue.CounterClockwise_Positive;
     private static final SensorDirectionValue STEER_ENCODER_DIRECTION = SensorDirectionValue.CounterClockwise_Positive;
-    private static final AbsoluteSensorRangeValue STEER_ENCODER_RANGE = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
+    private static final double STEER_ENCODER_RANGE = 0.5;
     private static final NeutralModeValue
             DRIVE_MOTOR_NEUTRAL_MODE_VALUE = NeutralModeValue.Brake,
             STEER_MOTOR_NEUTRAL_MODE_VALUE = NeutralModeValue.Brake;
     private static final double
-            DRIVE_SLIP_CURRENT = RobotHardwareStats.isSimulation() ? 1000 : 100,
-            STEER_CURRENT_LIMIT = RobotHardwareStats.isSimulation() ? 1000 : 50;
+            DRIVE_SLIP_CURRENT = RobotHardwareStats.isSimulation() ? 1000 : 80,
+            STEER_CURRENT_LIMIT = RobotHardwareStats.isSimulation() ? 1000 : 30;
     private static final double
             STEER_MOTOR_P = RobotHardwareStats.isSimulation() ? 75 : 75,
             STEER_MOTOR_I = 0,
-            STEER_MOTOR_D = 0;
+            STEER_MOTOR_D = 0,
+            STEER_MOTOR_KV = 1,
+            STEER_MOTOR_KA = 1;
     private static final double
             DRIVE_MOTOR_P = RobotHardwareStats.isSimulation() ? 50 : 50,
             DRIVE_MOTOR_I = 0,
-            DRIVE_MOTOR_D = 0;
+            DRIVE_MOTOR_D = 0,
+            DRIVE_MOTOR_KS = RobotHardwareStats.isSimulation() ? 1 : 0,
+            DRIVE_MOTOR_KV = RobotHardwareStats.isSimulation() ? 1 : 0,
+            DRIVE_MOTOR_KA = RobotHardwareStats.isSimulation() ? 1 : 0;
     static final boolean ENABLE_FOC = true;
     static final TalonFXConfiguration
             DRIVE_MOTOR_CONFIGURATION = generateDriveConfiguration(),
             STEER_MOTOR_CONFIGURATION = generateSteerConfiguration();
     static final CANcoderConfiguration STEER_ENCODER_CONFIGURATION = generateSteerEncoderConfiguration();
 
-    private static final double
-            DRIVE_MOMENT_OF_INERTIA = 0.03,
-            STEER_MOMENT_OF_INERTIA = 0.003;
     private static final int
             DRIVE_MOTOR_AMOUNT = 1,
             STEER_MOTOR_AMOUNT = 1;
     private static final DCMotor
             DRIVE_MOTOR_GEARBOX = DCMotor.getKrakenX60Foc(DRIVE_MOTOR_AMOUNT),
             STEER_MOTOR_GEARBOX = DCMotor.getFalcon500Foc(STEER_MOTOR_AMOUNT);
 
-    static final double WHEEL_DIAMETER_METERS = RobotHardwareStats.isSimulation() ? 0.1016 : 0.049149 * 2;
+    static final SysIdRoutine.Config DRIVE_MOTOR_SYSID_CONFIG = new SysIdRoutine.Config(
+            Units.Volts.of(5).per(Units.Second),
+            Units.Volts.of(8),
+            Units.Second.of(1000)
+    );
+
+    static final double WHEEL_DIAMETER_METERS = RobotHardwareStats.isSimulation() ? 0.1016 : 0.049274 * 2;
```

Comment:
add a way to set this for each module individually. look more at Triumph's swerve code.